src/AddTodoForm.js

```diff
@@ -1,27 +1,35 @@
-import React from 'react'; 
+import React, { useState } from 'react';
 
-const AddTodoForm = (props)=> { 
-    const handleAddTodo = (event)=>{ 
+const AddTodoForm = ({onAddTodo}) => {
+    const [todoTitle, setTodoTitle] = useState()
+    const handleTitleChange = (event) => {
+        let newTodoTitle = event.target.value;
+        setTodoTitle(newTodoTitle)
+    }
+    const handleAddTodo = (event) => {
         // created handler function
         event.preventDefault()
         // preventing default behavior of form submit 
-        let todoTitle =event.target.title.value;
-        // this variable holds the value of the input. 
         console.log(todoTitle);
-       event.target.title.value =' ' ;
-    //    clearing the input value after submit
+        onAddTodo({
+            id: Date.now(),
+            title: todoTitle,
+        })
+        setTodoTitle('')
+        // this is so the input is a blank string after typing
```

Comment:
nit: the general convention is to put comments above the line of code that they refer to